packages/BetterPhpDocParser/Printer/RemoveNodesStartAndEndResolver.php

```diff
@@ -26,9 +26,14 @@ public function resolve(PhpDocNode $originalPhpDocNode, PhpDocNode $currentPhpDo
         $lastEndPosition = null;
 
         foreach ($removedChildNodes as $removedChildNode) {
-            /** @var StartAndEnd $removedPhpDocNodeInfo */
+            /** @var StartAndEnd|null $removedPhpDocNodeInfo */
             $removedPhpDocNodeInfo = $removedChildNode->getAttribute(PhpDocAttributeKey::START_AND_END);
 
+            // it's not there when comment block has empty row "\s\*\n"
+            if (is_null($removedPhpDocNodeInfo)) {
```

Comment:
You can check instanceof check:

```suggestion
            if (! $removedPhpDocNodeInfo instanceof StartAndEnd) {

Reply:
That was how my first change looked, but phpstan did not like it
![image](https://user-images.githubusercontent.com/20553410/158116445-c0260163-b5ce-4428-824f-85d88c968f85.png)

Reply:
probably cache